tests/search/metrics/metrics.rb

```diff
@@ -83,6 +83,23 @@ def test_metrics
     assert_document_db_attribute_memory_usage(metrics)
   end
 
+  def test_metrics_imported_attributes
+    set_description("Check reporting of metrics with imported attributes")
+    deploy_app(SearchApp.new.cluster(SearchCluster.new('test').
+                      sd(selfdir + "parent.sd", { :global => true }).
+                      sd(selfdir + "child.sd").
+                      visibility_delay(0.001).
+                      tune_searchnode({:summary => {:store => {:cache => { :maxsize => 8192,
```

Comment:
Is this tuning really needed for the test? If not, please remove. Applies for visibility_delay() as well.